src/session.rs

```diff
@@ -631,11 +631,13 @@ where
         }
 
         // make chrome comply with w3c
-        cap.entry("goog:chromeOptions".to_string())
-            .or_insert_with(|| Json::Object(serde_json::Map::new()))
-            .as_object_mut()
-            .expect("goog:chromeOptions wasn't a JSON object")
-            .insert("w3c".to_string(), Json::from(true));
+        if cap.get("browserName") == Some(&Json::from("chrome")) {
```

Comment:
Since no other browser has a problem with this, and not everyone passes in `browserName` as a capability, I'd like to instead special-case Internet Explorer here. Only if the browser name specifically is internet explorer should we _not_ include this. Makes sense?

Reply:
I agree with you.
I think `goog:chromeOptions` is only for chrome, but only IE driver does not accept this capability.

Reply:
I’m sorry for the late comment. I fixed it.